app/gui/src/project-view/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -428,10 +428,7 @@ const CustomDropdownItemsKey: unique symbol = Symbol.for('WidgetInput:CustomDrop
 function isHandledByCheckboxWidget(parameter: SuggestionEntryArgument | undefined): boolean {
   return (
     parameter?.tagValues != null &&
-    arrayEquals(Array.from(parameter.tagValues).sort(), [
-      'Standard.Base.Data.Boolean.Boolean.False',
-      'Standard.Base.Data.Boolean.Boolean.True',
-    ])
+    arrayEquals(Array.from(parameter.tagValues).sort(), ['False', 'True'])
```

Comment:
Well, technically FQNs are better, because user could define their own False and True atoms.

But let it be for now.